### ISSUE TITLE
Define a way to improve NER installation through adding new cases and retraining it

### ISSUE THREAD

**Issue by danielkornev** [deepmipt/DeepPavlov#1307]
Want to contribute to DeepPavlov? Please read the [contributing guideline](http://docs.deeppavlov.ai/en/master/devguides/contribution_guide.html) first.

**What problem are we trying to solve?**:
```

```

**How can we solve it?**:
```

```

**Are there other issues that block this solution?**:
```

```

**Comment by acriptis** [comment]
I suppose a good NER Service should cover the following use cases through web interface:
1. Annotate requests with NER entites (as any NER tool must do) via REST API
2. Allow Admins to view log of requests for annotation and produced results
3. Allow Admins to add new training cases into the dataset (which will fine-tune NER model instantly or via button).

It seems to be a quite simple web app.

**Comment by Rajmehta123** [comment]
I agree. The 3rd point seems to be interesting. But, we cant expect the model to fine-tune in real-time. Did you mean transfer learning where we transfer the weights of the new data to the existing trained model?
